ui/src/app/pages/homepage/recent-resources.tsx

```diff
@@ -86,24 +87,27 @@ export const RecentResources = (fp.flow as any)(
     const {contentRect, measureRef} = this.props;
     const {offset, resources, loading} = this.state;
     const limit = (contentRect.client.width - 24) / 224;
-    return <div ref={measureRef} style={{display: 'flex', position: 'relative', minHeight: 247}}>
-      <div style={{display: 'flex', position: 'relative', alignItems: 'center',
-        marginTop: '-1rem', marginLeft: '-1rem', paddingLeft: '1rem', opacity: loading ? 0.5 : 1}}>
-        {resources.slice(offset, offset + limit).map((resource, i) => {
-          return <div key={i}> {this.createResourceCard(resource)} </div>;
-        })}
-        {offset > 0 && <Scroll
-          dir='left'
-          onClick={() => this.setState({offset: offset - 1})}
-          style={{position: 'absolute', left: 0, paddingBottom: '0.5rem'}}
-        />}
-        {offset + limit < resources.length && <Scroll
-          dir='right'
-          onClick={() => this.setState({offset: offset + 1})}
-          style={{position: 'absolute', right: 0, paddingBottom: '0.5rem'}}
-        />}
+    return (resources !== null && resources.length > 0) || loading ? <React.Fragment>
```

Comment:
nit: I think we may want to start formatting our multi-line ternary expressions like this:
The above can be rewritten as the following to improve readability and more clearly delineate the parts:
```
var foo = bar > baz ?
    value1 :
    value2;
```
https://eslint.org/docs/rules/multiline-ternary

I'd be happy to add this to the linter as well if this is something we want to be consistent about.

Reply:
If we are adding to linter, let's do as a separate PR. Meanwhile, style change applied :D